core/amber/src/main/scala/edu/uci/ics/texera/workflow/WorkflowCompiler.scala

```diff
@@ -28,13 +26,11 @@ class WorkflowCompiler(
       logicalPlan: LogicalPlan,
       logicalOpsToViewResult: List[String],
       errorList: Option[ArrayBuffer[(OperatorIdentity, Throwable)]]
-  ): PhysicalPlan = {
+  ): (PhysicalPlan, Set[GlobalPortIdentity]) = {
```

Comment:
add document about the returned values. it is not clear to me.

Reply:
Added documentation.